features/world-location-news.feature

```diff
@@ -82,12 +82,6 @@ Feature: World location news for people local to countries
     When I view the world location "British Antarctic Territory"
     Then I should see the publication "Penguins have rights too"
 
-  Scenario: View priorities for an international delegation
-    Given an international delegation "United Nations" exists
-    And a published worldwide priority "Oil field exploitation" exists relating to the international delegation "United Nations"
-    When I view the international delegation "United Nations"
-    Then I should see the worldwide priority "Oil field exploitation"
```

Comment:
Again, this is removed because we don't want them being displayed in metadata anymore.